### ISSUE TITLE
generic html and body styling

### ISSUE THREAD

**Issue by MiikaL** [h5p/h5p-editor-php-library#50]
Would it be possible to remove the generic html and body styling from the application.scss file? When using the editor as a component in an existing application, this kind of generic styling (particularly the body max-width) can mess up the larger application. Could it just be removed entirely, or then put behind a .h5p-editor class?

**Comment by porcospino** [comment]
This is related, so I'll comment here.

`editor/styles/css/application.css` and `editor/styles/scss/application.scss` both define `max-width: 960px` for the `html` and `body` elements.  This arbitrarily restricts the width of the editor on larger screens.

**Comment by icc** [comment]
That is correct, we hope to improve this as we improve the editor experience. Currently, editors like the Branching Scenario overrides this to make use of all the space.
The problem if we just remove this is that all the regular text fields will become way too big due to their default 100% width and this is not desirable in terms of UX.
